### PR TITLE
Feature/form add

### PR DESCRIPTION
**Pull request by iva585**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,14 @@
...
-    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <link rel="icon" type="image/svg+xml" href="favicon.ico" />
+    <link rel="icon" type="image/png" href="android-chrome-512x512.png" />
+    <link rel="icon" type="image/png" href="android-chrome-192x192.png" />
+    <link rel="icon" type="image/png" href="favicon-32x32.png" />
+    <link rel="icon" type="image/png" href="favicon-16x16.png" />
+    <link rel="icon" type="image/png" href="apple-touch-icon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite App</title>
+    <title>Hamiti</title>
```
Comment (Torfeltorfel) [inline index.html:12]: good catch!

File: src/app/components/RecipeForm/RecipeForm.tsx [hunk 2]
```diff
@@ -148,7 +173,7 @@ export default (): JSX.Element => {
         <TagList tags={recipeData.tags} />
       </Box>
       <Box sx={{ p: 2 }}>
-        <Button variant="outlined" color="error">
+        <Button onClick={handleCancel} variant="outlined" color="error">
```
Comment (Torfeltorfel) [inline src/app/components/RecipeForm/RecipeForm.tsx:176]: cool that you are using button variants!

**Review by derjubu — APPROVED** [review]
LGTM 🏅

**Review by Torfeltorfel — APPROVED** [review]
I impressed how you implemented redux and are able to use it!

**Pull request merged**